notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>@MichaelClifford do you want to also add this dataset <code>testgrid_291.json.gz</code> to the data folder?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I think  a better solution might be to change this to run this on the `testgrid_810.json.gz` (better to not have a bunch of example datasets) I think I had just generated a new one while going through the notebooks a while back. @oindrillac when you ran it did the last cell output a value greater than 0? i.e., were there flakes in that dataset?

Reply:
<p>Yes, that sounds reasonable. yes,  I got this <code>0.003037</code></p><p> </p>